Makefile

```diff
@@ -95,9 +95,17 @@ help-extended: #HELP Display extended help.
 #SECTION Development
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) #HELP Run golangci linter.
+lint: lint-custom $(GOLANGCI_LINT) #HELP Run golangci linter.
 	$(GOLANGCI_LINT) run --build-tags $(GO_BUILD_TAGS) $(GOLANGCI_LINT_ARGS)
 
+.PHONY: custom-linter-build
+custom-linter-build:
+	cd ./hack/ci/custom-linters/cmd/ && go build -o ../../../../bin/custom-linter
```

Comment:
```suggestion
	go build ./hack/ci/custom-linters/cmd/ -o ./bin/custom-linter
```

Reply:
I tried, but it did not work well. 
If we do not run the build from the, dir that fails with the go mod import.

Reply:
I will create a comment for this one to make it easier you to compare the latest state and suggestions